addons/auth_totp_portal/views/templates.xml

```diff
@@ -11,25 +11,55 @@
                         <i class="fa fa-warning"/>
                         Two-factor authentication not enabled
                     </div>
-                    <button type="button" class="btn btn-secondary" id="auth_totp_portal_enable">
-                        Enable two-factor authentication
-                    </button>
                 </t>
                 <t t-else="">
-                    <span class="text-success">
-                        <i class="fa fa-check-circle"/>
-                        Two-factor authentication enabled
-                    </span>
-                    <button type="button" class="btn btn-link" id="auth_totp_portal_disable">
-                        (Disable two-factor authentication)
-                    </button>
+                    <div class="form-group">
+                        <span class="text-success">
+                            <i class="fa fa-check-circle"/>
+                            Two-factor authentication enabled
+                        </span>
+                    </div>
                 </t>
+                <div class="form-group">
+                    <span>
+                        Two-factor authentication
+                    </span>
+                    <t t-if="not user_id.totp_enabled">
+                        <div class="o_boolean_toggle custom-control custom-checkbox ml-4" name="auth_totp_portal_toggle">
+                            <input type="checkbox" id="auth_totp_portal_enable" class="custom-control-input"/>
+                            <label for="auth_totp_portal_enable" class="custom-control-label">
+                                &#8203;
+                                <i class="fa fa-times-circle"></i>
+                            </label>
+                        </div>
+                    </t>
+                    <t t-else="">
+                        <div class="o_boolean_toggle custom-control custom-checkbox ml-4" name="auth_totp_portal_toggle">
+                            <input type="checkbox" id="auth_totp_portal_disable" class="custom-control-input" t-attf-checked="checked"/>
+                            <label for="auth_totp_portal_disable" class="custom-control-label">
+                                &#8203;
+                                <i class="fa fa-check-circle"></i>
+                            </label>
+                        </div>
+                    </t>
+                </div>
+                <div class="form-group">
+                    <span class="oe_grey">
+                        Two-factor authentication ("2FA") is a system of double authentication. The first one is your usual
+                        password and the second one is a code you get from a phone app or a sms.
+                    </span>
+                </div>
+                <div class="form-group">
+                    <span class="oe_grey">
+                        Odoo adivses you to install a 2FA phone based authenticator like Authy on your phone.
```

Comment:
Typo here.
But from what I understand, this whole thing will be moved to another PR (David) ?